test/test-options-cluster.js

```diff
@@ -0,0 +1,42 @@
+// Copyright IBM Corp. 2015. All Rights Reserved.
+// Node module: strong-supervisor
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+var CPUS = require('os').cpus().length;
+var options = require('../lib/options');
+var tap = require('tap');
+
+tap.test('cluster option validation', function(t) {
+  t.test('valid values', function(t) {
+    check(['node', 'sl-run', '--cluster=10'], 10);
+    check(['node', 'sl-run', '--cluster', '-10'], -10);
+    check(['node', 'sl-run', '--cluster', 'CPU'], CPUS);
+    check(['node', 'sl-run', '--cluster=cpu'], CPUS);
+    t.end();
+
+    function check(argv, expect) {
+      var o = options.parse(argv);
+      t.equal(o.cluster.size, expect);
```

Comment:
FWIW, `t.equal(_.get(o, 'cluster.size'), expect);` is the lodash approach you're thinking of.